modules/services/nix-gc.nix

```diff
@@ -95,8 +36,7 @@ in
 
           On Linux this is a string as defined by {manpage}`systemd.time(7)`.
 
-          On Darwin it must be one of: ${toString darwinIntervals}, which are
-          implemented as defined in the manual page above.
+          $lib.hm.darwin.{intervalDocumentation}
```

Comment:
Unless I'm missing something, this should be:
```suggestion
          ${lib.hm.darwin.intervalDocumentation}
```

Reply:
Yeah, that's my bad for missing that...

Reply:
sorry for this. I will fix it.

Reply:
Fixed by https://github.com/nix-community/home-manager/pull/7071.